omniport/core/open_auth/views/retrieve_data.py

```diff
@@ -100,6 +103,10 @@ def get(self, request):
             response_data['person']['display_picture'] = \
                 get_display_picture(person)
 
+        if 'residential_information.residence' in app_data_points:
+            response_data['residential_information']['residence'] = \
+                person.residentialinformation.residence.name
+
```

Comment:
We don't need to handle this case separately. Adding the following in `omniport/core/open_auth/constants/data_points.py` should work as intended
```
RESIDENCE = 'residential_information.residence.name'
```

Reply:
Did the changes